### ISSUE TITLE
Keep Focus Within Modals with Shift + Tab

### ISSUE THREAD

**Issue by jpdevries** [jpdevries/eureka#144]
I made sure when you blur the last focusable item in a modal it sets focus back to the first focusable item, but this doesn't work for Shift + Tab.
